src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/RequestHandler.java

```diff
@@ -509,11 +516,12 @@ private AssigningResult(final IRequestResolver resolver, final List<IToken<T>> c
         }
 
         @Override
-        public int compareTo(@NotNull final AssigningResult o)
+        public int compareTo(@NotNull final AssigningResult<T> o)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1210)